admin/bootstrap.py

```diff
@@ -95,6 +105,37 @@ def checkenv(args: argparse.Namespace) -> None:
         sys.exit(1)
 
 
+def is_missing_dependency() -> bool:
+    """
+    Check if there are any missing apt dependencies.
+    This applies to existing Tails systems where `securedrop-setup` may not have been
+    run recently.
+    """
+    # apt-cache -q0 policy $dependency1 $dependency2 $dependency3 | grep "Installed: (none)"
+    apt_query = f"apt-cache -q0 policy {APT_DEPENDENCIES_STR}".split(" ")
+    grep_command = ["grep", "Installed: (none)"]
```

Comment:
@zenmonkeykstop if the admin workstation has all the right dependencies, `is_missing_dependency` won't prompt for sudo or do any network calls.

If a package name changes (eg sq-keyring-linter gets replaced by sq), this still won't try and install sq-keyring-linter, because the apt-cache policy results wont be `Installed: (none)`, they'll be `N: unable to locate package $funnyname`.

If you're not a fan, let me know and we can at least move the check into the branch of envsetup where no existing venv is detected.